specificatie/filter.yaml

```diff
@@ -28,7 +28,7 @@ components:
       type: string
       pattern: ^[a-zA-Z0-9À-ž \.\-\']{1,200}$|^[a-zA-Z0-9À-ž \.\-\']{3,199}\*{1}$
       example: "Vries"
-    StraatFilter:
```

Comment:
Is dit correct? Moet er alleen worden gefilterd met de officiele naam? De zoek-met-straatnaam-huisnummer-en-gemeente-van-inschrijving.feature beschrijft filteren op straatnaam en naam openbare ruimte

Reply:
Inderdaad niet terecht. Volgens mij wil je echt bieden om te zoeken in straat én openbare ruimte

Reply:
Aangepast